### ISSUE TITLE
Form elements: Provide a clear indication of fields in error for information that is submitted (CAL-AMP21) 

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#13531]
**monday.com sync:** #10822002218

### Violation: 
 Provide a clear indication of fields in error for information that is submitted 

### WCAG Reference: 
- [3.3.1 Error Identification](https://www.w3.org/WAI/WCAG22/Understanding/error-identification) 

### Recommended solution
_TBD_, a spike is needed to determine how we can provide support similar to how native elements are communicating error handling/messaging.

**Background**:
- When form fields are submitted in error their context should be communicated with the following information:
    - [ ] Name of the field with the error
    - [ ] The error message 

### Severity: 
 High 

### Media Type: 
 Forms 

--- 
### Areas for Remediation: 
- Title: CAL-AMP21 Provide a clear indication of fields in error for information that is submitted 
- Module: M3: Form validation_Calcite Feedback Survey 
- Finding ID: [CAL-AMP21](https://esri.hub.essentia11y.com/short-link/qR1OgAEG1NOBj6YO) 

**Issue**
There are "Please fill out this field" errors that do not mention the name of the field in error. 

**User Impact**
Screen reader users and users with cognitive disabilities will have difficulty determining that these messages are errors instead of general instructions on how to fill the fields. They will also have difficulty determining which error corresponds to which field.

**Code Reference** 
 ```
<calcite-input-message aria-live="polite" id="inputValidationMessage" calcite-hydrated-hidden="false" scale="m" status="invalid" icon="" calcite-hydrated=""><!--?lit$985399698$-->Please fill out this field.</calcite-input-message>

```

**Steps to Reproduce**
1. Locate the error.
2. Notice that the error text does not include the name of the field(s) in error.
 
**Recommendation**
Provide a clear indication of fields in error for information that is submitted. Include the name of the field in the error message.

**Compliant Code Example**
```
<!-- example error message that identifies the error and field in error -->
Error: The following errors were detected in the form:
<ul>
<li><a href="#fname">The First name field is empty, it is a required field and must be filled in.</a>
</li>
</ul>
 … 
<label for="fname">*First Name:</label> <input type="text" required id="fname" aria-describedby="fname_error"> <span id="fname_error">Error: First name is required.</span>
```

 **Issue Image** 
 https://desktopqa/aic_lap/Calcite/Calcite2025/CAL-AMP21.png 
 

 --- 
### Additional Resources: 
- [LA Platform Report Link](https://esri.hub.essentia11y.com/short-link/qR1OgAEG1NOBj6YO) 
- [Esri Accessibility Resources](https://esriis.sharepoint.com/sites/Accessibility)
- Report Source: Product accessibility evaluation conducted on Calcite by Level Access and delivered August, 2025.

**Comment by DitwanP** [comment]
~~We think this could be part of the `ElementInternals` work on~~ #8126. 

On further inspection, this is not directly reliant on the `ElementInternals` work but would benefit from being tackled after that work is complete.
